src/Microsoft.Data.Analysis/PrimitiveDataFrameColumn.cs

```diff
@@ -833,5 +833,29 @@ protected internal override Delegate GetValueGetterUsingCursor(DataViewRowCursor
         {
             return GetGroupedOccurrences<T>(other, out otherColumnNullIndices);
         }
+
+        public override PrimitiveDataFrameColumn<bool> ElementwiseIsNull()
+        {
+            var ret = new BooleanDataFrameColumn(Name, Length);
+
+            for (long i = 0; i < Length; i++)
+            {
+                ret[i] = !_columnContainer[i].HasValue;
+            }
+
+            return ret;
+        }
+
+        public override PrimitiveDataFrameColumn<bool> ElementwiseIsNotNull()
+        {
+            var ret = new BooleanDataFrameColumn(Name, Length);
+
+            for (long i = 0; i < Length; i++)
+            {
+                ret[i] = _columnContainer[i].HasValue;
```

Comment:
Please us `IsValid` instead of `_columnContainer[i].HasValue`

Reply:
Fixed